tests/python/pytorch/sparse/test_sparse_matrix.py

```diff
@@ -450,6 +450,146 @@ def test_has_duplicate():
     assert csc_A.has_duplicate()
 
 
+@pytest.mark.parametrize("dense_dim", [None, 4])
+@pytest.mark.parametrize("row", [(0, 1, 1, 2, 3, 4), (0, 1, 2, 3, 4, 4)])
+@pytest.mark.parametrize("col", [(0, 2, 4, 3, 5, 0), (1, 2, 5, 3, 0, 1)])
+@pytest.mark.parametrize("dim", [0, 1])
+@pytest.mark.parametrize("index", [(0, 1, 4), (1, 1, 3)])
+def test_index_select(dense_dim, row, col, dim, index):
```

Comment:
Please follow https://github.com/dmlc/dgl/blob/f281959ac70fb50a96a8b243133c9d013ccd3e9f/tests/python/pytorch/sparse/test_matmul.py#L32 to test different sparse formats. https://github.com/dmlc/dgl/blob/f281959ac70fb50a96a8b243133c9d013ccd3e9f/tests/python/pytorch/sparse/utils.py#L139 also provides utility function to convert a sparse matrix to a pytorch dense tensor.

Reply:
I have modified the code. However, I find there are some problems with current DGL COO slice func, that is, it dose not support deplicate select ids: https://github.com/dmlc/dgl/blob/master/src/array/cpu/spmat_op_impl_coo.cc#L799-L820 
It uses a hashmap to store ids and scans rows once, which is different with the implementation of CSR slice: 
https://github.com/dmlc/dgl/blob/master/src/array/cpu/spmat_op_impl_csr.cc#L384-L486

Reply:
That is a problem. You can only support slicing on CSR/CSC for now. If we think slicing on COO is important, we can implement it by sparse native operators.

Reply:
OK, I just remove the coo test, since it supports neither duplicate ids nor cuda operations.